public/js/showoff.js

```diff
@@ -1554,6 +1554,34 @@ function nextStep(updatepv)
   }
 }
 
+function prevSec(updatepv)
+{
+  $(currentSlide).find('video').each(function() {
+    console.log('Pausing videos on ' + currentSlide.attr('id'))
+    $(this).get(0).pause();
+  });
+
+  var curSec = currentSlide.attr('data-section');
```

Comment:
There is already getAllSections and getCurrentSections functionality, up at 955 and 965.  I like this curSec search better than getCurrentSections logic, but would rather refactor the existing function and use it here.

Reply:
Ack. I didn't find these helper functions.

Reply:
Just tried `getAllSections()` in my console and it returned an empty array. `getCurrentSections()` returned an empty object.

Reply:
It seems `getAllSections` returns `div.notes-section`, which are not the same sections I think.

Reply:
No, they're poorly named, tbh. What they do is return the different kind of notes sections you can have, or have on the current slide. In our courseware we have `notes` and `handouts`, but you can add arbitrarily named sections and they'll show up as tabs in the notes area.

Reply:
Right, so that doesn't return the sections in the slides, which do not appear to be listed anywhere. It seems they're simply built dynamically on line 505.